source/scenario/scenario_dummy.vwf.yaml

```diff
@@ -0,0 +1,221 @@
+# Copyright 2014 Lockheed Martin Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may 
+# not use this file except in compliance with the License. You may obtain 
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software 
+# distributed under the License is distributed on an "AS IS" BASIS, 
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and 
+# limitations under the License.
+
+---
+extends: source/scenario/scenario.vwf
+properties:
+  scenarioName: scenario_dummy
+
+  startState:
+  - setProperty:
+    - rover3
+    - visible
+    - true
+  - setProperty:
+    - rover2
+    - visible
+    - true
+  - setProperty:
+    - rover
+    - battery
+    - 9001
+  - setProperty:
+    - rover
+    - batteryMax
+    - 9001
+  - callMethod:
+    - rover
+    - allowedBlocksChanged
```

Comment:
This is an event that will be fired by the setting of `blockly_allowedBlocks`. It should not be done explicitly.